torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -204,6 +204,10 @@ auto handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyOb
  *  precedence.
  *
  *  'obj' is an object to check for a __torch_function__ implementation
+ * 
+ * If changing this file, please report the benchmarks in
+ * 'benchmarks/overrides_benchmark'. See the instructions in the 
+ * 'README.md' in that directory.
```

Comment:
I don't think *all* changes to the argument parser should require running the benchmarks.

Reply:
Agreed, can change to "If changing this file in a way that may affect the __torch_function__ overhead"